yt/loaders.py

```diff
@@ -1569,7 +1536,6 @@ def load_sample(
         )
 
     kwargs = {**specs["load_kwargs"], **kwargs}
```

Comment:
@neutrinoceros , you wanna rename this `kwargs` as well? e.g., 
```suggestion
    load_kwargs = {**specs["load_kwargs"], **kwargs}
```
(or whatever name we decide on in the other comment thread)

Reply:
It would make sense, sure !